tools/docusaurus-to-antora-conversion-scripts/convert-docs.sh

```
@@ -0,0 +1,87 @@
+  echo "Usage: ./your_script.sh /path/to/your/source_directory"
+  exit 1
+fi
+
+OUTPUT_DIRECTORY="../modules"
+
+# Create the output and partials directories if they don't exist
+mkdir -p "$OUTPUT_DIRECTORY"
+
+function remove_leading_tabs() {
+  local mdx_file="$1"
+  local content="$(cat "$mdx_file")"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)